spec/canvas_link_migrator/imported_html_converter_spec.rb

```diff
@@ -37,6 +37,13 @@
       expect(bad_links).to be_nil
     end
 
+    it "converts a wiki reference with migration id" do
+      test_string = %(<a href="%24WIKI_REFERENCE%24/wiki/A?query=blah">Test Wiki Page</a>)
```

Comment:
I feel just a tad weird that this is wiki instead of pages, since "wiki" theoretically references all of the wiki pages.  Are the URLs we're using actually "wiki" urls?  It looks like in your changed code it has "pages".

Reply:
Oops you're right. Thanks Mysti!